### ISSUE TITLE
Amun integration test fails for ocp-test

### ISSUE THREAD

**Issue by goern** [thoth-station/integration-tests#25]
**Describe the bug**
```
    Then wait for inspection to finish successfully                               # features/steps/amun_api.py:96 1.112s
      Traceback (most recent call last):
        File "/home/goern/.local/share/virtualenvs/integration-tests-Me6nOCO7/lib/python3.6/site-packages/behave/model.py", line 1329, in run
          match.run(runner.context)
        File "/home/goern/.local/share/virtualenvs/integration-tests-Me6nOCO7/lib/python3.6/site-packages/behave/matchers.py", line 98, in run
          self.func(context, *args, **kwargs)
        File "features/steps/amun_api.py", line 104, in wait_for_inspection_to_finish
          response = get_inspection_status(context.amun_api_host, context.inspection_id)
        File "/home/goern/.local/share/virtualenvs/integration-tests-Me6nOCO7/lib/python3.6/site-packages/amun/lib.py", line 47, in get_inspection_status
          api_response = api_instance.get_inspection_status(inspection_id)
        File "/home/goern/.local/share/virtualenvs/integration-tests-Me6nOCO7/lib/python3.6/site-packages/amun/swagger_client/amun/inspection_api.py", line 485, in get_inspection_status
          return self.get_inspection_status_with_http_info(inspection_id, **kwargs)  # noqa: E501
        File "/home/goern/.local/share/virtualenvs/integration-tests-Me6nOCO7/lib/python3.6/site-packages/amun/swagger_client/amun/inspection_api.py", line 567, in get_inspection_status_with_http_info
          collection_formats=collection_formats)
        File "/home/goern/.local/share/virtualenvs/integration-tests-Me6nOCO7/lib/python3.6/site-packages/amun/swagger_client/api_client.py", line 343, in call_api
          _preload_content, _request_timeout, _host)
        File "/home/goern/.local/share/virtualenvs/integration-tests-Me6nOCO7/lib/python3.6/site-packages/amun/swagger_client/api_client.py", line 182, in __call_api
          return_data = self.deserialize(response_data, response_type)
        File "/home/goern/.local/share/virtualenvs/integration-tests-Me6nOCO7/lib/python3.6/site-packages/amun/swagger_client/api_client.py", line 254, in deserialize
          return self.__deserialize(data, response_type)
        File "/home/goern/.local/share/virtualenvs/integration-tests-Me6nOCO7/lib/python3.6/site-packages/amun/swagger_client/api_client.py", line 293, in __deserialize
          return self.__deserialize_model(data, klass)
        File "/home/goern/.local/share/virtualenvs/integration-tests-Me6nOCO7/lib/python3.6/site-packages/amun/swagger_client/api_client.py", line 631, in __deserialize_model
          kwargs[attr] = self.__deserialize(value, attr_type)
        File "/home/goern/.local/share/virtualenvs/integration-tests-Me6nOCO7/lib/python3.6/site-packages/amun/swagger_client/api_client.py", line 293, in __deserialize
          return self.__deserialize_model(data, klass)
        File "/home/goern/.local/share/virtualenvs/integration-tests-Me6nOCO7/lib/python3.6/site-packages/amun/swagger_client/api_client.py", line 633, in __deserialize_model
          instance = klass(**kwargs)
        File "/home/goern/.local/share/virtualenvs/integration-tests-Me6nOCO7/lib/python3.6/site-packages/amun/swagger_client/models/inspection_status.py", line 58, in __init__
          self.build = build
        File "/home/goern/.local/share/virtualenvs/integration-tests-Me6nOCO7/lib/python3.6/site-packages/amun/swagger_client/models/inspection_status.py", line 81, in build
          raise ValueError("Invalid value for `build`, must not be `None`")  # noqa: E501
      ValueError: Invalid value for `build`, must not be `None`
```

**To Reproduce**
1. clone this repo
2. run `./test.sh`

/assign @bissenbay 
/king bug
/priority critical-urgent

**Comment by bissenbay** [comment]
I think these 2 PRs should fix: https://github.com/thoth-station/amun-client/pull/137 and https://github.com/thoth-station/integration-tests/pull/26

**Comment by goern** [comment]
looks like the fix has not been deployed: https://github.com/thoth-station/thoth-application/commit/434a422f9fbff310d3677800821f0e6f2560ad3a

**Comment by goern** [comment]
even with `X-Amun-Version: 0.7.2+storage.0.24.4.common.0.14.2` being deploy the test fails, pls double check. thx!

**Comment by harshad16** [comment]
The new release of amun-api was scheduled based on this , now version v0.7.3 should have the content of pr 475 and other update PR's.  This should fix the issue.

**Comment by bissenbay** [comment]
This should fix: https://github.com/thoth-station/integration-tests/issues/27

**Comment by bissenbay** [comment]
> This should fix: #27

@goern @fridex @harshad16 I think this is solved because I run `./test.sh` and `Feature: Aggregating Build Logs` has passed.

**Comment by fridex** [comment]
Feel free to close this if all is fine.
